modules/readpeakBidAdapter.md

```diff
@@ -4,7 +4,7 @@ Module Name: ReadPeak Bid Adapter
 
 Module Type: Bidder Adapter
 
-Maintainer: kurre.stahlberg@readpeak.com
+Maintainer: tuomo.tilli@readpeak.com
```

Comment:
I would recommend setting this to an internal alias so that you don't have to change it and can include multiple people in case one is not available

Reply:
That's a good idea thanks! Updated.